### ISSUE TITLE
DEC maps for DWI workflow

### ISSUE THREAD

**Issue by araikes** [nipreps/mriqc#1223]
### What would you like to see added in this software?

Following on from mention in #1131, DEC maps would provide visual diagnostics the need for axis flips/swaps in the b-table. 

### Do you have any interest in helping implement the feature?

Yes, but I would need guidance

**Comment by araikes** [comment]
Looks like a CFA file is already being generated and just not used.

**Comment by oesteban** [comment]
> Looks like a CFA file is already being generated and just not used.

Yup. Would it be okay to just drop the FA and render the color FA? cc/ @arokem @yasseraleman

**Comment by arokem** [comment]
Yeah - I think that would be fine!
